scripts/constants.py

```diff
@@ -15,16 +15,16 @@
 import sys
 
 # Kubernetes branch to get the OpenAPI spec from.
-KUBERNETES_BRANCH = "release-1.15"
+KUBERNETES_BRANCH = "release-1.16"
 
 # client version for packaging and releasing.
-CLIENT_VERSION = "11.0.0-snapshot"
+CLIENT_VERSION = "12.0.0-snapshot"
 
 # Name of the release package
 PACKAGE_NAME = "kubernetes"
 
 # Stage of development, mainly used in setup.py's classifiers.
-DEVELOPMENT_STATUS = "4 - Beta"
+DEVELOPMENT_STATUS = "3 - Alpha"
```

Comment:
nit: we have been using Beta in the master branch (x.0.0-snapshot) meaning a "dev" status. It doesn't have any real effect since we don't publish the snapshot client. Perhaps Alpha fits it better. I think we can stick with one and don't need to toggle between Alpha and Beta in the future

Reply:
I was researching what should be the classifier for an upstream development branch, but couldn't find any concrete precedence for our use case.

I thought settling for Alpha would make more sense than Beta.

> I think we can stick with one and don't need to toggle between Alpha and Beta in the future
:+1: